scripts/package-nuget.ps1

```diff
@@ -145,7 +145,7 @@ $NugetSourceFolder = Join-Path $RootDir "src/distribution"
 if ($UWP) {
     $PackageName = "Microsoft.Native.Quic.MsQuic.UWP.$Tls"
 } elseif ($XDP) {
-    Copy-Item -Path (Join-Path $PSScriptRoot xdp-devkit.json) -Destination (Join-Path $PackagingDir xdp-devkit-temp.json)
+    Copy-Item -Path (Join-Path $PSScriptRoot xdp.json) -Destination (Join-Path $PackagingDir xdp-temp.json)
```

Comment:
Why we would still call it -temp?

Reply:
Why was it called temp to begin with?

Reply:
I remember because you said this file was temporary until we have an official XDP release so we can remove it or something.

Reply:
Why do we still need this file with XDP 1.0? I assume now that XDP is official, XDP will guarantee backcompat and we will never need to know which XDP version MsQuic is built on?

Reply:
That's a fair question. I think more formalized dependency/package managers tend to check in the precise version they were built against (lock files) even if the logical dependency is less strict, to ensure repeatable builds. Otherwise yes I agree, QUIC should be able to depend on 1.x.